### DISCUSSION TITLE
Grandchildren Description missing

### DISCUSSION THREAD

**Discussion by kroening-stroeer** [openooh/venue-taxonomy#41] [Q&A] [answered]
Hi all,

great work here! [Ströer ](https://www.stroeer.de/nc/aussenwerbung/werbemedien/kategorie/public-video/werbemedien/Product/list.html) will most likely adopt the new venue-taxonomy early 2021.
Since we sell a lot of mall assets as well I´d like to ask you for a more specific feedback on the mall grandchildren:

Grandchild Category | Category Definition | Enumeration ID | String Value |  Stroeer explanation
Concourse | TBC | 20501 | retail.malls.concourse | These screens are in hallways and at the escalators
Food Court | TBC | 20502 | retail.malls.food_court | These screens are in hallways at food courts
Spectacular | TBC | 20503 | retail.malls.spectacular | it this something like Ströer [Digital Dream](https://www.stroeer.de/nc/aussenwerbung/werbemedien/kategorie/public-video/produkt/digital-dream.html) ?

**Comment by jayshao** [comment, marked as answer]
@kroening-stroeer I think your notes are consistent in general with the intent of the spec. I'll see if maybe the notes got accidentally dropped in the transcription process, but I suspect the original editors may have just run out of steam a bit as they got further down the document...

We're working towards a 1.1 update, and I'll see if we can make sure to take another copy edit through the document. If you have any suggestions or points of improvement (or want to participate more closely) now is a good time to look over the issues :)

The "Spectacular" item I think is... hard to define exactly. I think in most respects it's something along the lines of "large, impactful, reaches a lot of people" or maybe more simply "might headline a media plan" - though I think that's a... loose description. I (personally) would feel comfortable with your defining that screen in such a way, given it's prominence and limited surrounding distractions especially - but it might be worth trying to get more points of view.

**Reply by jayshao** [reply]
Logged as https://github.com/openooh/venue-taxonomy/issues/110 since I *think* we took a pass in 1.1 - but to make sure we normalize in 1.2
